### DISCUSSION TITLE
Bro，Question about pretrained model

### DISCUSSION THREAD

**Discussion by Youskrpig** [open-mmlab/mmyolo#701] [Q&A] [answered]
老哥，我想问下比如yolov5的backbone部分Cspdarknet53在imagenet上的预训练权重有么？如果我在我自己的数据集上验证效果，只load backbone在imagenet上的权重和直接load mmyolo整个模型在coco上的预训练权重差距会很大么？

**Comment by hhaAndroid** [comment, marked as answer]
@Youskrpig 暂时没有提供耶。因为本身会训练 300e，预训练权重的意义不是特别大。
